keyboards/uzu42/rev1/rev1.c

```diff
@@ -0,0 +1,32 @@
+#include "uzu42.h"
+
+
+#ifdef AUDIO_ENABLE
+    float tone_startup[][2] = SONG(STARTUP_SOUND);
+    float tone_goodbye[][2] = SONG(GOODBYE_SOUND);
+#endif
+
+#ifdef SSD1306OLED
+void led_set_kb(uint8_t usb_led) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+    //led_set_user(usb_led);
+}
+#endif
+
+void matrix_init_kb(void) {
+
+    #ifdef AUDIO_ENABLE
+        _delay_ms(20); // gets rid of tick
+        PLAY_SONG(tone_startup);
+    #endif
+
+	matrix_init_user();
+};
+
+void shutdown_kb(void) {
+    #ifdef AUDIO_ENABLE
+        PLAY_SONG(tone_goodbye);
+      	_delay_ms(150);
+      	stop_all_notes();
+    #endif
+}
```

Comment:
This entire block should be removed, as this is now handled in the core.  Having this here will override user settings, actually. 

https://docs.qmk.fm/#/feature_audio?id=audio